windowsforms/helloworld-links/README.md

```diff
@@ -0,0 +1,12 @@
+# Hello-World with Links
+
+This sample shows how to share code between a NetCore3 and NetFramework projects. This approach allows to reuse the same code files, this is important because allows to use the designers from the NetFramework project and reuse it from NetCore.
```

Comment:
- a NetCore3 and NetFramework projects --> either one of:
   - .NET Core 3.0 and .NET Framework projects 
   - a .NET Core 3.0 and .NET Framework project
- allows to reuse  --> allows you to reuse
- period instead of comma after files
- this is important because allows to --> This is important because it allows you to
- NetFramework --> .NET Framework
- and reuse it from NetCore --> and reuse them in .NET Core